content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Internal Telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+The Collector offers multiple ways to measure and monitor its own health. In
+this section, you'll learn how to enable internal observability.
+
+## Enabling observability internal to the Collector
+
+By default, the Collector exposes service telemetry in two ways:
+
+- Internal [metrics](#configuring-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configuring-logs) are emitted to `stdout`.
+
+[Traces](#configuring-traces) are not exposed by default but two feature gates
+offer experimental support for a configuration based on the OpenTelemetry
+Configuration schema.
```

Comment:
```suggestion
## Activate internal telemetry in the Collector

By default, the Collector exposes its own telemetry in two ways:

- Internal [metrics](#configuring-metrics) are exposed using a Prometheus
  interface which defaults to port `8888`.
- [Logs](#configuring-logs) are emitted to `stdout` by default.

[Traces](#configuring-traces) are not exposed by default. Two feature gates
offer experimental support for a configuration based on the OpenTelemetry
Configuration schema.
```

I wonder if we should mention traces at all. How would they help in troubleshooting Collector issues? Perhaps @bogdandrutu and @open-telemetry/collector-maintainers know.

Reply:
We could leave traces out of the picture for now, given that the support is quite experimental yet. @codeboten What do you think?

Reply:
I've made the text changes suggested by @theletterf and will wait to hear from @codeboten about removing the traces section.

Reply:
Pinging @codeboten one more time to see if they have thoughts before I move forward with removing the traces section.

Reply:
Removed traces from this PR in c2fef88. Resolving this conversation.